javascript/javascript2/week3/Game/script.js

```diff
@@ -0,0 +1,72 @@
+const btn = document.getElementById("startBtn");
+let inputTime = document.getElementById('inputTime');
+const timer = document.getElementById("timer");
+
+const scoreS = document.getElementById("scoreS");
+const scoreL = document.getElementById("scoreL");
+
+const divForS = document.getElementById("right");
+const divForL = document.getElementById("left");
+
+const wonSkey = document.getElementById("wonSkey");
+const wonLkey = document.getElementById("wonLkey");
+
+let startNUmS = 0;
+let startNUmL = 0;
+
+btn.addEventListener("click", () => {
+    let StartingSec = inputTime.value;
+    ended = false;
+    startGame()
+    timer.innerHTML = `${StartingSec} : 00`;
+    const countdown = setInterval(() => {
+        StartingSec--;
+        timer.innerHTML = `${StartingSec} : 00`;
+        if (StartingSec <= 0) {
+            ended = true;
+            clearInterval(countdown);
+            endGame();
+
+        }
+    }, 1000);
+})
+function startGame() {
+    document.addEventListener("keypress", (event) => {
```

Comment:
Also it would be nice to remove this event listener after the game ended.

Reply:
Yes i will update it